.github/workflows/linux.yml

```diff
@@ -193,7 +193,7 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-20.04-16-cores
```

Comment:
We could probably run VLM tests in a separate job, to keep using default runners for simpler tests in future.